consensus/enclave/Cargo.toml

```diff
@@ -47,3 +47,6 @@ mc-util-metrics = { path = "../../util/metrics" }
 aes-gcm = "0.10.1"
 lazy_static = "1"
 sha2 = "0.10"
+
+[patch.crates-io]
+mc-sgx-core-types = { git = "https://github.com/dolanbernard/sgx.git", rev = "7003bd505bb60bd85840020908d75d9590b5665b" }
```

Comment:
All of these patches will be removed before this PR is merged. They are just here until the SGX repo changes are merged and released.

Reply:
patches look to be removed now :+1: